resources/src/main/java/org/robolectric/res/Fs.java

```diff
@@ -119,10 +119,6 @@ static String baseNameFor(Path path) {
   }
 
   public static InputStream getInputStream(Path path) throws IOException {
-    // otherwise we get ClosedByInterruptException, meh
-    if (path.toUri().getScheme().equals("file")) {
```

Comment:
Because the code inside the `if` block can be replaced with:
```java
return new BufferedInputStream(Files.newInputStream(path));
``

which is the same as the `else` clause.